packages/design-system/src/components/OcResource/OcResource.vue

```diff
@@ -56,6 +56,7 @@
           :full-path="resource.path"
           :is-path-displayed="isPathDisplayed"
           :is-extension-displayed="isExtensionDisplayed"
+          @click="$emit('clickFolder')"
```

Comment:
Why do we need `click.stop` on the parent folder link but omit the `stop` modifier here?

Reply:
I checked that, there is no need for the stop on both 🥇